src/extensions.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "name": "subgraph",
+    "description": "This Scaffold-ETH 2 extension helps you build and test subgraphs locally for your contracts. It also enables interaction with the front-end and facilitates easy deployment to Subgraph Studio.",
+    "repository": "https://github.com/scaffold-eth/create-eth-extensions",
```

Comment:
Isn't this field called `github` in scaffoldeth.io? 

https://github.com/scaffold-eth/scaffoldeth.io/blob/8f3818f4cc94d0e7e6c24116ecea0c385078a571/packages/nextjs/components/ExtensionCard.tsx#L43